frontend/src/components/CcfriEstimator.vue

```diff
@@ -532,8 +532,8 @@
                                 <span
                                   :style="
                                     'color:' +
-                                    (active ? 'white' : 'white') +
```

Comment:
Can we do some cleanup of the styles on both month sliders while we're in there? We don't need to refactor to classes (I suspect they are styles for the sake of precedence)  but the following would help:
- Remove custom font-family. All fonts should be the default BCSans
- Remove conditional statements that have no effect? E.g. `:color="isSelected ? '#FFFFFF' : '#FFFFFF'"`, isSelected ? 'white' : 'white'

Reply:
Additional frontend changes:
- Removed unnecessary font color conditional statements.
- Removed unnecessary font-family attributes as it's already inherited from .v-application parent class.